solr/core/src/test/org/apache/solr/update/TestInPlaceUpdatesDistrib.java

```diff
@@ -1146,7 +1146,9 @@ private void reorderedDBQsResurrectionTest() throws Exception {
       // the resurrection can happen from there (instead of the leader)
       update.setParam(
           DistributedUpdateProcessor.DISTRIB_FROM,
-          ((HttpSolrClient) NONLEADERS.get(1)).getBaseURL());
+          ((HttpSolrClient) NONLEADERS.get(1)).getBaseURL()
+              + "/"
+              + NONLEADERS.get(1).getDefaultCollection());
```

Comment:
this actually helps you understand what we are doing in the DISTRIB_FROM by seeing the call to `getDefaultCollection`.